managed/devops/bin/yb_platform_util_py3.py

```diff
@@ -0,0 +1,483 @@
+# Copyright (c) YugaByte, Inc.
+#
+# Copyright 2021 YugaByte, Inc. and Contributors
+#
+# Licensed under the Polyform Free Trial License 1.0.0 (the "License"); you
+# may not use this file except in compliance with the License. You
+# may obtain a copy of the License at
+#
+# https://github.com/YugaByte/yugabyte-db/blob/master/licenses/POLYFORM-FREE-TRIAL-LICENSE-1.0.0.txt
+
+import urllib.request
+import json
+from urllib.error import HTTPError
+
+
+# def exception_handling():
+
+
+def exception_handling(func):
+    def inner_function(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except HTTPError as e:
+            content = e.read().decode("utf-8")
+            print (content)
```

Comment:
This script is a user-friendly script. Print statement return data to script and later script print that response to the Terminal. Because of that reason we had used print.

Reply:
Sure, but stderr will be visible to end-user as well, but this allows to filster it out for automation then.